packages/client/src/stores/notification.js

```diff
@@ -52,6 +52,7 @@ const createNotificationStore = () => {
           message,
           icon,
           dismissable: !autoDismiss,
+          duration,
```

Comment:
`duration` isn't used, so we don't need to store it here. We only need it down below in the `setTimeout` block.